.github/workflows/ansible-release.yml

```
@@ -0,0 +1,109 @@
+name: ansible-release
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+env:
+  CI_COMMIT_MESSAGE: 'Ansible "${{ inputs.ansible_version }}": Dependencies, changelog and porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+    name: 'Build Ansible community distribution (${{ inputs.ansible_version }})'
+    runs-on: ubuntu-latest
+
+
+    steps:
+      - name: Check out antsibull
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull
+          ref: main
+          path: antsibull
+
+      - name: Check out antsibull-core
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull-core
+          ref: main
+          path: antsibull-core
+
+      - name: Check out antsibull-changelog
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull-changelog
+          ref: main
+          path: antsibull-changelog
+
+      - name: Pre-create build directory
+        run: mkdir -p antsibull/build
+
+# We need to checkout the docs repo here
+
+      # This is where the antsibull build-release role expects it by default
+      - name: Check out ansible-build-data under antsibull build directory
+        uses: actions/checkout@v3
+        with:
+          path: antsibull/build/ansible-build-data
+          ref: main
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        working-directory: antsibull
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install ansible-core . ../antsibull-core ../antsibull-changelog
```

The release workflow should install antsibull from PyPI:

```suggestion
          python3 -m pip install --upgrade pip
          python3 -m pip install ansible-core antsibull
```